src/common/vector/value_vector.cpp

```diff
@@ -427,6 +428,9 @@ void StringVector::addString(ValueVector* vector, uint32_t vectorPos, ku_string_
     if (ku_string_t::isShortString(srcStr.len)) {
         dstStr.setShortString(srcStr);
     } else {
+        if (srcStr.len > BufferPoolConstants::PAGE_256KB_SIZE) {
+            throw RuntimeException(ExceptionMessage::overLargeStringValueException(srcStr.len));
```

Comment:
codecov says this is not covered, is it a false positive?

Reply:
I guess only two of the addString variants are covered by the long string tests. I'm not sure how to trigger the others via an end-to-end test.

Reply:
This one seems to be used by addBlob, which is used in the rdf reader.

The other one that's missed seems to be used in `encode_function.h` and `decode_function.h`